changelog/1609.txt

```diff
@@ -0,0 +1,5 @@
+```release-note:feature
```

Comment:
This could probably just be an improvement :-)

Reply:
done